### ISSUE TITLE
Mac failure after Analyse

### ISSUE THREAD

**Issue by ColinRobbins** [ColinRobbins/scm-helper#14]
**Describe the bug**
On A Mac 1.3 fails after the analyse stage

**Additional context**
```
Analysing...
Done.
Traceback (most recent call last):
  File "./scm-cmd.py", line 15, in <module>
    main.cmd()
  File "/Users/gb043435/development/scm-helper/scm_helper/main.py", line 274, in cmd
    output = issues.print_by_error(reports)
  File "/Users/gb043435/development/scm-helper/scm_helper/issue.py", line 543, in print_by_error
    res += print_dict(self.by_error, report)
  File "/Users/gb043435/development/scm-helper/scm_helper/issue.py", line 617, in print_dict
    for xissue in sorted(xdict[key1][key2], key=lambda x: x[0]):
TypeError: '<' not supported between instances of 'NoneType' and 'NoneType'
```

**Comment by ColinRobbins** [comment]
Fixed in 1.3.1

**Comment by ColinRobbins** [comment]
Fixed in the Dev tree.
